include/ableton/Link.hpp

```diff
@@ -359,21 +359,31 @@ class Link
       bool isPlaying, std::chrono::microseconds time, double beat, double quantum);
 
   private:
-    friend Link;
+    friend BasicLink<Clock>;
     link::ApiState mOriginalState;
     link::ApiState mState;
     bool mbRespectQuantum;
   };
 
 private:
+  using Controller = ableton::link::Controller<
```

Comment:
Please delete the now unused type alias in https://github.com/Ableton/link/blob/aa477048017a4d4fa28922bab33da3bdb3c2dabf/include/ableton/platforms/Config.hpp#L79

Reply:
OK

Reply:
Done.